starter-code/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>REVERA Clone</title>
+  <!--  Default browser-styles resetting -->
+  <link href="thirdparty/reset.css" type="text/css" rel="stylesheet">
+  <!--  Bootstrap CSS file -->
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css" rel="stylesheet">
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!--  Bootstrap JS file -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <!--  Main CSS file-->
+  <link href="main.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+  <div class="container-fluid">
+    <header>
+      <h1>REVERA</h1>
+      <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Sample Page</li>
+        <li>Portfolio</li>
+        <li>Blog</li>
+        <li>Parent Page</li>
+        <li>Contact</li>
+      </ul>
+    </header>
+    <section class="row main-pic">
+      <img src="img/project-2.jpg" alt="cat pic">
+    </section>
+    <section class="row features">
+      <article class="col-sm-4">
+        <i class="glyphicon glyphicon-glass"></i>
+        <h2>Web design</h2>
+        <p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius</p>
+      </article>
+      <article class="col-sm-4">
+        <i class="glyphicon glyphicon-star"></i>
+        <h2>Web development</h2>
+        <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat luctus.</p>
+      </article>
+      <article class="col-sm-4">
+        <i class="glyphicon glyphicon-heart"></i>
+        <h2>User interface</h2>
+        <p>Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien dignissim, eu dapibus mi molestie. Suspendisse dictum convallis quam et sodales. Praesent non enim et magna congue gravida.</p>
+      </article>
+    </section>
+    <article class="row latest-projects">
+      <h2>LATEST PROJECTS</h2>
+      <h3>FEW OF THE LASTEST PORTFOLIO ITEMS</h3>
+      <hr>
+    </article>
+    <section class="row project-widgets">
+      <article class="col-sm-3 col-xs-6">
+        <img src="img/project-1.jpg" alt="project 1 screenshot">
+        <p>Nunc libero elit, iaculis nec augue non</p>
+      </article>
+      <article class="col-sm-3 col-xs-6">
```

Comment:
This is nice responsive design :)
However, notice that it may be easier on the eye leaving some side margins in this section, as in the original.